samples/nextjs/src/pages/[[...path]].tsx

```diff
@@ -0,0 +1,93 @@
+import { NextPage, GetStaticPaths, GetStaticProps } from 'next';
+import Error from 'next/error';
+import Layout from '../components/Layout';
+import { SitecoreContext } from '@sitecore-jss/sitecore-jss-nextjs';
+import { SitecorePageProps, extractPath } from '../lib/page-props';
+import componentFactory from '../temp/componentFactory';
+import { configBasedLayoutService as layoutService } from '../lib/layout-service';
+import { configBasedDictionaryService as dictionaryService } from '../lib/dictionary-service';
+
+const SitecorePage: NextPage<SitecorePageProps> = (props) => {
+  const { layoutData } = props;
+
+  if (!layoutData?.sitecore?.route) {
+    // layoutData will be missing for an invalid path
+    return <Error statusCode={404} />
+  }
+
+  const context = {
+    route: layoutData.sitecore.route,
+    itemId: layoutData.sitecore.route?.itemId,
+    ...layoutData.sitecore.context
+  };
+
+  return (
+    <SitecoreContext componentFactory={componentFactory} context={context}>
```

Comment:
@ambrauer Current implementation will not work with `SitecoreContext` from JSS 15. Is it fine for now?

Reply:
Correct, the latest JSS 15 `SitecoreContext` seems to be fine (no longer need the override we had in the POC which was using JSS 14).